README.md

```diff
@@ -315,6 +315,41 @@ The following example contains two custom udev rules that will create `/etc/udev
 }
 ```
 
+#### fan
+
+An object that defines thermal related configuration. Available for Jetson Orin devices running Jetpack 6 or newer, balenaOS v6.1.24 or newer and Supervisor v16.10.0 or newer.
```

Comment:
I realize I am late for this - it really bothers me seeing that we are breaking the separation between meta-balena and device repositories. imo, the feature should have been designed generic even if only the Jetson Orin device is actually using it, and this mention to Jetson Orin should be removed. It sets a bad precedent and will make it more confusing to maintain the OS in future.

Also, balenaOS and supervision version are implied - the field is supported from the version where the README is updated. I really think adding these details is a bad practise.

@klutchell

Reply:
Maybe there is a way we can avoid creating a "special butterfly" situation out of this which is what orin will become.